docker-compose.tests.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  backend_test:
+    build: backend
+    env_file:
+      - .envs/.local/.backend
+    entrypoint: ["pytest", "--ds=blogchain.settings.test"]
+    volumes:
+     - ./backend:/app
```

Comment:
Should tests use volumes?

Reply:
Maybe you can add dir one time, so tests won't impact your file system.

Reply:
or make it read only? correct me if it's wrong idea

Reply:
@Mc01 ^

Reply:
Read only should be fine 😉

Reply:
Generally they shouldn't use volumes to avoid unneeded dependencies but I use them here because I'm lazy. Instead of running `docker-compose up` on every change in the tests I just run `bash` inside of a container and run the entrypoint manually whenever I do something (thanks to that I don't need to wait until it sets the containers up every time).

I may as well remove that volume and just copy the files inside the container.